softwarereview_intro.pt.Rmd

```diff
@@ -0,0 +1,86 @@
+# Revisão por pares de software, por quê? Por quê? {#softwarereviewintro}
+
+```{block, type="summaryblock"}
+Este capítulo contém uma [introdução geral] (#whatissoftwarereview) ao nosso sistema de revisão por pares de software para pacotes, [razões para enviar um pacote] (#whysubmit), [razões para se voluntariar como revisor] (#whyreview), [por que nossas revisões são abertas] (#whyopen) e agradecimentos aos [atores do sistema] (#editores e revisores).
+
+Nosso sistema foi recentemente ampliado para [revisão por pares de software estatístico] (https://ropensci.org/stat-software-review/).
+
+_Se você usar nossos padrões/checklists etc. ao revisar um software em outro lugar, informe aos destinatários (por exemplo, editores de periódicos, estudantes, revisão interna de código) que eles vieram da rOpenSci e nos informe em [nosso fórum público] (https://discuss.ropensci.org/c/usecases) ou [em particular por e-mail] (https://ropensci.org/contact/). 
+```
+
+## O que é a revisão por pares do software rOpenSci? {#whatissoftwarereview}
+
+A equipe do rOpenSci [da rOpenSci](https://ropensci.org/packages/) é parcialmente contribuído por membros da equipe e parcialmente contribuído por membros da comunidade, o que significa que o conjunto resulta de uma grande diversidade de habilidades e experiências dos desenvolvedores. Como garantir a qualidade de todo o conjunto? É aí que entra em cena a revisão por pares de software: os pacotes contribuídos pela comunidade passam por um processo de revisão transparente, construtivo, não contraditório e aberto. Para esse processo, você depende principalmente do trabalho voluntário, [editores associados](#associateditors) gerenciam o fluxo de entrada e garantem o andamento dos envios; os autores criam, enviam e aprimoram seus pacotes; [revisores](#reviewers) dois por envio, examinam o código do software e a experiência do usuário. [Esta publicação no blog](https://www.numfocus.org/blog/how-ropensci-uses-code-review-to-promote-reproducible-science/) escrito por editores da rOpenSci é uma boa introdução à revisão por pares de software da rOpenSci. Você pode encontrar outras postagens de blog sobre a própria revisão e os pacotes revisados [através da tag "software-peer-review" no blog da rOpenSci](https://ropensci.org/tags/software-peer-review/).
+
+Você pode reconhecer os pacotes do rOpenSci que foram revisados por pares por meio de um emblema verde "peer-reviewed" em seu README, com links para suas revisões (cf [este exemplo](https://github.com/ropensci/restez#locally-query-genbank-)); e por meio de um ícone de comentário azul próximo à sua descrição em [página de pacotes do rOpenSci](https://ropensci.org/packages/) com links para as revisões.
```

Comment:
About the part "ícone de comentário azul próximo à sua descrição":
It was quite difficult to understand what it meant in the translation.
I looked up on the website and this is an attempt to describe it a bit better.
But this is a small change from the source book, I'm not sure how to proceed.

```suggestion
Você pode reconhecer os pacotes da rOpenSci que foram revisados por pares por meio de um selo (*badge*) verde com o texto "peer-reviewed" em seu README, com links para suas revisões ([por exemplo](https://github.com/ropensci/restez#locally-query-genbank-)); e por meio de um botão azul com o texto *"Peer-reviewed"* próximo à sua descrição na [página de pacotes da rOpenSci](https://ropensci.org/packages/) com links para as revisões.
```

Reply:
You can do this kind of changes. We also did it in some part for the Spanish version.